.github/CONTRIBUTING.md

```diff
@@ -1,135 +1,63 @@
+- Make sure the bug hasn’t already been fixed. Use the latest snapshot of libGDX to try to reproduce your issue.
+- Create a concise, self-contained example to demonstrate the issue. If we can’t reproduce your issue, we can’t fix it.
 
-Guidelines for bug reports:
+Once you have done this, [create a new issue](https://github.com/libgdx/libgdx/issues/new) in the project and follow the instructions in the template that is shown to help us triage your issue.
 
-1. **Use the GitHub issue search** &mdash; check if the issue has already been
-   reported. Make sure to search all issues, not only the open issues.
+### Pull Requests
 
-2. **Check if the issue has been fixed** &mdash; try to reproduce it using the
-   latest `master` or development branch in the repository.
+PRs for libGDX are always welcome and can be a quick way to get your fix or improvement slated for the next release. In general, PRs should:
 
-3. **Isolate the problem** &mdash; create a [reduced test
-   case](https://libgdx.com/wiki/articles/getting-help#executable-example-code).
+- Only fix/add _one_ thing.
+- Add a meaningful description that explains what the PR does, what might still be missing (maybe testing on certain platforms?) and anything else potential reviewers need to know.
 
-A good bug report shouldn't leave others needing to chase you up for more
-information. Please try to be as detailed as possible in your report. What is
-your environment? What steps will reproduce the issue? What browser(s) and OS
-experience the problem? What would you expect to be the outcome? All these
-details will help people to fix any potential bugs.
+Before starting to contribute, please also give a quick look at our [Contributing Guidelines](https://libgdx.com/dev/contributing/#guidelines), in particular the sections on the Contributor License Agreement and Formatting. For changes that address core functionality or would require breaking changes, it's best to discuss your proposal first on our [Discord](https://libgdx.com/community/discord/) server (check the `#libgdx-contributions` channel). 
 
-If you have found a bug and want to fix it yourself immediately, great!
-Create a [pull request](#pull-requests) with your proposed correction and
-a description of the problem you are fixing. Please do **not** create a separate
-issue for the bug report, the pull request is enough.
+Now, let's see how you can actually create a PR:
 
-See [Getting Help](https://libgdx.com/wiki/articles/getting-help) for more information and an example.
+1.  Fork the repository to your own Github account. Clone the project to your machine. Set up the project in your local IDE. These steps are detailed over on [our website](https://libgdx.com/dev/from-source/#setting-the-project-up).
+2.  Create a branch locally with a succinct but descriptive name
```

Comment:
I'm not sure whether it is just me, but I had to look up what "succinct" means. Maybe it is better to use something else? "concise"?

Reply:
It's a good word! I think "concise" is better by meaning, in my view, "short and accurate," whereas "succinct" means "complete while being as short as possible," as far as I know. [Succinct is still a good word to be aware of.](https://en.wikipedia.org/wiki/Succinct_data_structure)